composer.json

```diff
@@ -89,13 +90,13 @@
         "sonata-project/classification-bundle": "^4.0",
         "sonata-project/doctrine-orm-admin-bundle": "^4.0",
         "symfony/browser-kit": "^4.4 || ^5.4 || ^6.0",
-        "symfony/filesystem": "^4.4 || ^5.4 || ^6.0",
```

Comment:
One last question, why this was removed ?

Reply:
It's already in the "require" section with the same versions. Does it makes sense to have a duplicate entry in require-dev? I can roll it back if it was there for a reason :)

Reply:
didn't see the require section. You were right to remove it then. thnaks